src/resources/js/aggregator-facebook-login.js

```diff
@@ -0,0 +1,43 @@
+var tribe_aggregator = tribe_aggregator || {};
+( function( $, _, ea ) {
+	'use strict';
+
+	ea.receive_message = function( event ) {
```

Comment:
I feel like this should be nestled under another level. Maybe `ea.facebook.receive_message`?